Everything/EverythingAPI.cs

```diff
@@ -123,11 +123,13 @@ public List<SearchResult> Search(string keyWord, CancellationToken token, SortOp
                 EverythingApiDllImport.Everything_SetSearchW(keyWord);
                 EverythingApiDllImport.Everything_SetOffset(offset);
                 EverythingApiDllImport.Everything_SetMax(maxCount);
-                
-                if(!EverythingApiDllImport.Everything_IsFastSort(sortOption))
-                {
+
+                var fastSortOptionEnabled = EverythingApiDllImport.Everything_IsFastSort(sortOption);
+                CheckAndThrowExceptionOnError();
```

Comment:
What does this method do?

Reply:
it checks the error thrown by Everything via API, it's how the code checks for service not running. The service not running error is 'silent'.

https://www.voidtools.com/support/everything/sdk/everything_isfastsort/, see EVERYTHING_ERROR_IPC